test/e2e/page-objects/pages/send/send-token-page.ts

```diff
@@ -60,11 +50,13 @@ class SendTokenPage {
     console.log('Send token screen is loaded');
   }
 
-  async fillRecipient(recipientAddress: string): Promise<void> {
-    console.log(
-      `Fill recipient input with ${recipientAddress} on send token screen`,
-    );
-    await this.driver.pasteIntoField(this.inputRecipient, recipientAddress);
+  async clickAssetPickerButton() {
+    await this.driver.clickElement(this.assetPickerButton);
+  }
+
+  async clickSecondTokenListButton() {
+    const elements = await this.driver.findElements(this.tokenListButton);
+    await elements[1].click();
   }
 
   async fillAmount(amount: string): Promise<void> {
```

Comment:
@chloeYue, the link takes to the file. Hope this message helps, this could cause race conditional and this is not part of this PR implementation.

Reply:
Thanks , this function indeed seems anti pattern, i will take a note and address it in separate PR.

Reply:
Thanks @chloeYue